ucsmsdk/ucshandle.py

```diff
@@ -235,7 +235,14 @@ def query_dns(self, *dns):
         if not dns:
             raise ValueError("Provide Comma Separated string of Dns")
```

Comment:
Change the Error string. "Provide Comma Separated string of Dns" -> "Provide Comma Separated string of Dns or List of Dns"